src/helper.js

```diff
@@ -158,6 +159,30 @@ function setHome(newPath) {
 	// Load theme color from the web manifest
 	const manifest = JSON.parse(fs.readFileSync(manifestPath, "utf8"));
 	this.config.themeColor = manifest.theme_color;
+
+	// log dir probably shouldn't be world accessible.
+	// Create it with the desired permission bits if it doesn't exist yet.
+	let logsStat = undefined;
+
+	try {
+		logsStat = fs.statSync(userLogsPath);
```

Comment:
can we not `fs.statSync(userLogsPath, { throwIfNoEntry: false })`? I'm confused by the comment in the catch

Reply:
we can't, that's a feature of v14.17.0 butour minimum version is 12